extensions/amp-analytics/amp-analytics.md

```diff
@@ -729,6 +729,10 @@ interval. Use `timerSpec` to control when this will fire.
   will be infinity.
 - `immediate` trigger timer immediately or not. Boolean, defaults to true
 
+[tip type="note"]
+The timer trigger will continue to send out requests regardless of document state (inactive or hidden), until the `maxTimerLength` has been reached or `stopSpec` has been met. In the case of no `stopSpec`, the `maxTimerLength` will default to infinity.
```

Comment:
`maxTimerLength` is only default to 2 hours if there's no `stopSpec`. it default to infinity when `stopSpec` exist.